### DISCUSSION TITLE
关于图片缩放的建议

### DISCUSSION THREAD

**Discussion by kwonder0926** [tamlok/vnote#1867] [Ideas]
现在的设置中，只设置放入图片的缩放比例，但是通常截取的图片大小不一，即便缩放后放进行，看着也是不整齐的。
所以我想建议可不可以添加一个固定图片大小的设置，使得每次粘贴的图片都默认成这个固定大小，例如500px。
这样文档中的图片右边都是对齐的，会很好看

**Comment by tamlok** [comment]
设置里面把这个勾上就好了：

![image](https://user-images.githubusercontent.com/5763157/131842592-ea1a19a7-ae39-4ed4-bca3-14f86e7c0743.png)

**Reply by kwonder0926** [reply]
谢谢您的回复
我勾选上且重启过了的，但是我每次往里粘贴不同大小的截图时，它都会出现不同的缩放宽度，而不是我所期望的固定宽度
我希望的固定宽度是，不论我粘贴什么尺寸的图片，它的[缩放宽度]处都是同一个默认值（eg. 500px）
![Snipaste_2021-09-02_21-54-09](https://gitee.com/kwonder/figurebed-typora/raw/master/images/Snipaste_2021-09-02_21-54-09.png)
